package.json

```diff
@@ -20,13 +20,9 @@
     "prepare": "husky install"
   },
   "dependencies": {
-    "@0xsequence/indexer": "^0.35.5",
-    "@0xsequence/multicall": "^0.35.5",
```

Comment:
we'll see how removal plays on staging for this one might have to add these back since lambda is weird